jhub_apps/hub_client/hub_client.py

```diff
@@ -136,8 +137,21 @@ def normalize_server_name(self, servername):
         # Max limit for servername is 255 chars
         return text[:240]
 
+    def _find_user_server(
+            self, servername
+    ) -> typing.Tuple[typing.Optional[dict], typing.Optional[dict]]:
```

Comment:
I think `user["name"]` is a string, not a dictionary:

```suggestion
    ) -> typing.Tuple[typing.Optional[str], typing.Optional[dict]]:
```

Personally I prefer using named tuples, but since this is a private method I think a normal tuple is fine.

Reply:
This was marked as resolved but I still see the typing using `dict` for the first argument, is this right?

Reply:
My bad, committed now.